src/sockets/sockets.ts

```diff
@@ -2314,53 +2328,37 @@ function updateRoomMaxPlayers(number) {
 
 // Functions to manage ranked and unranked queue
 
-// Unranked queue
-type MatchMakingQueueItem = {
-  id: string;
-  user: Object;
-  timeJoinedAt: Date;
-};
-
 let unrankedQueue6Players: MatchMakingQueueItem[] = [];
 let prospectivePlayersFor6UQ: MatchMakingQueueItem[] = [];
 let readyPlayersFor6UQ: MatchMakingQueueItem[] = [];
```

Comment:
Could you make this into a class please? It'll make it much cleaner by removing globals, easier to spin up a second ranked queue, and overall code quality will improve a lot :)